### DISCUSSION TITLE
Encontro #73 - Fluxo de Código e Debug

### DISCUSSION THREAD

**Discussion by endersonmenezes** [codaqui/institucional#101] [Encontros]
Arquivo do Encontro: [Clique aqui](https://replit.com/@menezesenderson/CoarseMiniatureParticle#script.js)

Nesse encontro conseguimos ver metade do conteúdo explicando sobre Debug e Fluxo de Código. Gostaria que vocês @codaqui/membros fizessem uma pequena pesquisa sobre **Algoritmo** e postassem um simples algoritmo utilizando **Pseudocódigo** de alguma atividade do dia a dia de vocês.

**Exemplo**:

_Insira sua pesquisa aqui_

---

**Pseudocódigo**:

```
CRIAR FUNCAO DESPERTAR:
  - Acordar
  - Levantar da cama
  - Tomar café
  - Escovar dentes
  - Trabalhar

INICIO;
SE HORARIO == 06:30:
    DESPERTAR();
```

**Comment by ArtumosGOC** [comment]
[O QUE É ALGORITIMO](https://tallos.com.br/blog/o-que-e-algoritmo-e-como-ele-funciona/#:~:text=Um%20algoritmo%20nada%20mais%20%C3%A9,instru%C3%A7%C3%B5es%20bastante%20simples%20e%20exatas.)

---

**Pseudocódigo**:

```
CRIAR FUNCAO TOMAR REMEDIO:
	- ABRIR POTAS DO ARMARIO
	- PEGAR O REMEDIO
	- ABRIR REMEDIO
	- TOMAR REMEDIO
	- FECHAR REMEDIO
	- GUARDAR NO ARMARIO
	- FECHAR ARMARIO
```

```
INCIO:
	
	ARMARIO.ABRIR;
	PEGA_r = PEGAR_ITEM("REMEDIO");
	PEGA_r.ABRIR;
	COMPRIMIDO_r = PEGA_r; 
	COMPRIMIDO_r = PEGAR_ITEM("COMPRIMIDO");
	TOMAR_ITEM(COMPRIMIDO_r);
	PEGA_r.FECHAR;
	ARMARIO = GUARDAR_ITEM("REMEDIO");
	ARMARIO.FECHAR;


	TOMAR_ITEM(TEXTO):
		TEXTO = PESSOA.PLANO_REAL.TOMAR;

	GUARDAR_ITEM(TEXTO):
		TEXTO = PESSOA.PLANO_REAL.GUARDAR;

	PEGAR_ITEM(TEXTO):
		TEXTO = PESSOA.PLANO_REAL.PEGAR;
```

**Reply by endersonmenezes** [reply]
Muito bom @ArtumosGOC ! Criou até um método, bem melhor que o meu =]
